src/bridge/handlers/authHandler.ts

```
@@ -0,0 +1,53 @@
+  payload: AuthPayload,
+): Promise<AuthResponse> => {
+  switch (type) {
+    case 'auth:login':
+      return handleLogin();
+    case 'auth:logout':
+      return handleLogout();
+  }
+};
+
+const handleLogin = async (): Promise<AuthResponse> => {
+  const { setAccessToken, setRefreshToken } = useAuthStore();
```

Biome (2.4.4)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)